tooling/google-analytics-baseline-checker/public/index.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<!--
+ Copyright 2025 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ -->
+<html lang="en-us">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Google Analytics Baseline Checker</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header class="Header">
+      <div class="Container">
+        <svg class="Header-logo" title="Baseline Widely available logo">
+          <use href="icons.svg#baseline-widely-available"></use>
+        </svg>
+        <h1 class="Header-title">
+          <span class="Header-titlePrefix">Google Analytics</span>
+          <span class="Header-titleSuffix">Baseline Checker</span>
+        </h1>
+        <p class="Header-subTitle">
+          A tool to help Google Analytics users determine the best Baseline
+          version for their site, <strong>based on their users</strong>.
```

Comment:
I know the accompanying web.dev article will cover this, but I also wonder if the tool itself should be more proactive in interpreting the results to suggest what the user's Baseline targets should be. Specifically:

- When do I target a Baseline year instead of WA?
- When is NA a good target?
- How do I pick the right Baseline year?

Maybe the demo could have a numeric slider that defaults to a support target like 98%, and automatically highlight the appropriate Baseline version.

Reply:
Yeah, I debated whether or not to include any commentary on how to select a Baseline version. I left it out mostly because I wasn't yet sure what was planned for @malchata's accompanying content. @malchata what do you think?

Reply:
I have a draft in the works to discuss exactly this problem. All the things @rviscomi laid out here are a part of what I'm looking to cover, and I think it will dovetail nicely into coverage about the tool proposed in this PR.